--- node_modules/@vuepress/shared-utils/lib/parseVueFrontmatter.js-org	2018-12-19 21:04:35.000000000 +0100
+++ node_modules/@vuepress/shared-utils/lib/parseVueFrontmatter.js	2018-12-19 21:04:40.694383401 +0100
@@ -15,7 +15,7 @@
 function parse(src) {
     const output = component_compiler_utils_1.parse({
         source: src,
-        compiler: vue_template_compiler_1.default,
+        compiler: vue_template_compiler_1,
         needMap: false
     });
     const find = output.customBlocks.find(block => block.type === 'frontmatter');
