--- node_modules/vuepress-plugin-sitemap/index.js.orig	2019-03-26 16:08:37.000000000 +0100
+++ node_modules/vuepress-plugin-sitemap/index.js	2019-03-28 09:04:03.373698922 +0100
@@ -50,8 +50,8 @@
       const pagesMap = new Map()
 
       pages.forEach(page => {
-        const lastmodISO = page.lastUpdated
-          ? new Date(page.lastUpdated).toISOString()
+        const lastmodISO = page.lastUpdatedTstamp
+          ? new Date(page.lastUpdatedTstamp).toISOString()
           : undefined
         const { normalizedPath } = stripLocalePrefix(page.path, localeKeys)
         const relatedLocales = localesByNormalizedPagePath.get(normalizedPath)
